File: src/main/java/ludwig/model/FunctionReferenceNode.java
package ludwig.model;

public class FunctionReferenceNode extends Node<FunctionReferenceNode> {
    @Override
    public <T> T accept(NodeVisitor<T> visitor) {
        return visitor.visitFunctionReference(this);
    }

    @Override
    public String toString() {
        return "ref";
    }
}

File: src/main/java/ludwig/model/FunctionReferenceNode.kt
package ludwig.model

class FunctionReferenceNode : Node<FunctionReferenceNode>() {
    override fun <T> accept(visitor: NodeVisitor<T>): T {
        return visitor.visitFunctionReference(this)
    }

    override fun toString(): String {
        return "ref"
    }
}
